pkg/gofr/datasource/surrealdb/surrealdb.go

```diff
@@ -673,6 +678,17 @@ func (c *Client) sendOperationStats(ql *QueryLog, startTime time.Time) {
 	ql.Namespace = c.config.Namespace
 	ql.Database = c.config.Database
 
+	c.metrics.RecordHistogram(context.Background(), "app_surrealdb_stats", float64(duration),
+		"namespace", ql.Namespace,
+		"database", ql.Database,
+		"operation", ql.OperationName)
+
+	if c.db != nil {
+		c.metrics.SetGauge("app_surrealdb_open_connections", 1)
+	} else {
+		c.metrics.SetGauge("app_surrealdb_open_connections", 0)
+	}
```

Comment:
🎨 nitpicking

I'm a strong advocate against using `else` statement, you know it 😅

What about this

```suggestion
	var nbConnection float64
	if c.db != nil {
		nbConnection = 1
	}
	c.metrics.SetGauge("app_surrealdb_open_connections", nbConnection)
```

or

```suggestion
	var nbConnection float64
	if c.db != nil {
		nbConnection++
	}
	c.metrics.SetGauge("app_surrealdb_open_connections", nbConnection)
```